models/stics/R/write.config.STICS.R

```diff
@@ -1274,62 +437,69 @@ write.config.STICS <- function(defaults, trait.values, settings, run.id) {
     str_ns <- site_id
   }
 
-  # I guess not important what this is called as long as it's consistent in usms
-  SticsRFiles::set_soil_txt(file = sols_file, param="typsol", value=paste0("sol", str_ns))
+  soils_df <- data.frame(soil_name = paste0("sol", str_ns))
```

Comment:
@qdbell @istfer Heads up that I resolved a merge conflict here -- the changes didn't actually overlap, but it was a good reminder that I should ask if my changes in #3428  work with your filename logic.